service/src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -66,20 +65,33 @@
   /** SQL query for reading all columns from the resource table */
   private static final String RESOURCE_SELECT_SQL =
       "SELECT workspace_id, cloud_platform, resource_id, name, description, "
-          + "stewardship_type, resource_type, cloning_instructions, attributes,"
+          + "stewardship_type, cloud_resource_type, resource_type, cloning_instructions, attributes,"
           + " access_scope, managed_by, associated_app, assigned_user, private_resource_state"
           + " FROM resource WHERE workspace_id = :workspace_id ";
 
   private static final RowMapper<DbResource> DB_RESOURCE_ROW_MAPPER =
       (rs, rowNum) -> {
+        // "notset" is the default value inserted when the resource_type column was created.
```

Comment:
how is "notset" better than `null`?

Reply:
If I specify a value, then I can add a not-null constraint. The constraint helps ensure that we properly fill in new resource records.